### PR TITLE
Fix for Autodoc build on ReadTheDocs

### LINKED ISSUE #1928: Autodoc not rendering API docs

### PR DESCRIPTION
**Pull request by derekpierre**
Fixes #1928

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -205,8 +206,36 @@ def remove_module_license(app, what, name, obj, options, lines):
...
+def run_apidoc(_):
+    from sphinx.ext import apidoc
+    source_dir = Path(__file__).parent.resolve()
+    nucypher_module_dir = source_dir.parent.parent
+
+    # Command: sphinx-apidoc [OPTIONS] -o <OUTPUT_PATH> <MODULE_PATH> [EXCLUDE_PATTERN …]
+    apidoc_command = []
+
+    # ---- execution options/paths ----
+    apidoc_command.extend(['-fME',
```
Comment (KPrasch) [inline docs/source/conf.py:218]: There's some extra indentation here

File: docs/source/conf.py [hunk 2]
```diff
@@ -205,8 +206,36 @@ def remove_module_license(app, what, name, obj, options, lines):
...
+                                        '-t', f'{source_dir / "apidoc"}',
+                                        '-o', f'{source_dir / "api"}',
+                                        f'{nucypher_module_dir}'])
+
+    # ---- exclusion patterns (*must be last to be added*) ----
+    # general patterns
+    apidoc_command.append('*conftest*')
+
+    # files/folders relative to `nucypher` project directory (results in required absolute paths)
+    exclusion_items = ['setup.py', 'tests', Path('nucypher', 'utilities'), 'scripts']
```
Comment (KPrasch) [inline docs/source/conf.py:228]: :+1:

File: docs/source/conf.py (hunk 1 above)
Comment (derekpierre) [reply docs/source/conf.py:218]: ✔️

**Comment by derekpierre** [comment]
>  Have you tried to run a test build on RTD?

@KPrasch I haven't - but with some guidance, I can. How can I test it on RTD?

**Comment by derekpierre** [comment]
@KPrasch, I was able to build it successfully using:
1. RTD Docker setup: https://github.com/readthedocs/readthedocs-docker-images/blob/master/CONTRIBUTING.rst
2. Build steps from https://readthedocs.org/projects/nucypher/builds/10965536/ with some minor modifications to parameters for the `sphinx-build` step

Once merged into `kerabatsos` it's easier for me to run an actual build on RTD itself since it is a `branch` of master, as opposed to from my own dev fork.

**Comment by derekpierre** [comment]
@KPrasch Successful build of `kerabatsos` on RTD: https://readthedocs.org/projects/nucypher/builds/10974606/

I enabled a build for `nucypher/kerabatsos` branch to test it, I'll delete the setting from RTD now since it was successful.

**Pull request merged**